src/index.css

```diff
@@ -1,8 +1,44 @@
 /* ./src/index.css */
+
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
-
 div.code {
-  white-space: pre;
+    white-space: pre;
+}
+
+@layer utilities {
+    .inner-shadow {
+        box-shadow: inset 0 -1px 0 0 rgb(255 255 255 / 10%);
+    }
+    .active-page {
+        position: absolute;
+        width: 0;
+        height: 2px;
+        bottom: 0;
+        left: 0;
+        background-color: #ff444f;
+        visibility: hidden;
+        transition: all .2s ease-in-out
+    }
+    .navigation-transition {
+        width: 20px;
+        visibility: visible;
+    }
+    .burger {
+        transition: left .4s ease 0s;
+    }
+    .transition-left {
+        left: 0;
+    }
+    .mobile-nav-after {
+        content: "";
+        position: absolute;
+        left: 0px;
+        width: 4px;
+        height: 40px;
+        background-color: red;
+        border-top-left-radius: 4px;
+        border-bottom-left-radius: 4px;
+    }
```

Comment:
is this the way how we can css to our custom classes where tailwind would not be enough?

Reply:
yes, It is one of the ways!
I picked this because it doesn't need any other third party packages.